docs/reference/kernel_language.rst

```diff
@@ -51,8 +51,7 @@ Supported ``__global__`` functions are:
   * Run on the device
   * Called (launched) from the host
 
-HIP ``__global__`` functions must have a ``void`` return type. The first parameter in a HIP ``__global__``
-function must have the type ``hipLaunchParm``. Refer to :ref:`kernel-launch-example` to see usage.
```

Comment:
"hipLaunchParm lp" also need be removed from 

__global__ MyKernel(hipLaunchParm lp,...)

in sample code.

Reply:
Removed